src/test/java/core/basesyntax/service/impl/DataParserServiceImplTest.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.exception.WrongOperationLetterException;
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DataParserServiceImplTest {
+    private static DataParserServiceImpl dataParserService;
+
+    @BeforeClass
+    public static void beforeClass() {

```

Comment:
```suggestion
    public static void init() {
```
don't forget that this is a method, naming of methods should start with verb, the same for other test classes

Reply:
fixed